taglib/mpeg/adtsheader.cpp

```diff
@@ -0,0 +1,207 @@
+/***************************************************************************
+    copyright           : (C) 2018 inMusic brands, inc. All rights reserved.
```

Comment:
This code has a conflicting license. It can't be "All rights reserved" at the same time as GPL.

Reply:
Good point. We will need @shaforostoff to make this change since he's the author.

Reply:
the web-site says LGPL and MPL. Anyways, I will remove that